content/price.json

```diff
@@ -47,5 +47,10 @@
       },
       "action": "Approve & Remove"
     }
+  },
+
+  "trade": {
```

Comment:
and I forgot the reasoning again behind using `Trade`, seems if we title it with `Swap tokens` we should also call the whole thing `Swap` down to the component names

Reply:
Ah, yes, I changed the names a few times. I took the trade from uniswap, the button that takes you to the swap interface is called "Trade"